src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3848,9 +3848,10 @@ const char* GfxGetOpcodeName(int8_t opcode) {
         } else {
             SPDLOG_CRITICAL("Unhandled OP code: 0x{:X}, for loaded ucode: {}", (uint8_t)opcode,
                             (uint32_t)ucode_handler_index);
-            return nullptr;
         }
     }
+
+    return nullptr;
 }
```

Comment:
Probably a good idea to also move the `SPDLOG_CRITICAL`.

Reply:
The meaning is slightly different, so I put a new log statement as an `else`